packages/prisma/migrations/20241119190500_update_event_type_translation/migration.sql

```diff
@@ -0,0 +1,14 @@
+-- AlterTable
+ALTER TABLE "EventTypeTranslation" RENAME COLUMN "id" TO "uid";
```

Comment:
@hbjORbj We can't rename from one column to another in a migration. This will run before the code deployment finishes and any query built by Prisma that references the "id" column will fail.

It needs to be done in a 2-step process like with `sourceLang` and `targetLang`

Reply:
addressed